README.md

```diff
@@ -136,8 +136,11 @@ df = DataFrame(A = 1:3, B = [2, 1, 2])
 nameA = :A
 df2 = @transform(df, C = :B - cols(nameA))
 
+nameA_string = "A"
```

Comment:
maybe make it a `julia> ....` and then print output (like we do in docstrings in DataFrames.jl)? It can be a separate PR of course later

Reply:
yeah I'm procrastinating some docs changes. I'm going to move things to a real documentation system soon so I won't be worried about the diffs.